app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Check this file and compare it to the given design. There are some differences, like the Terms & conditions text, or the image with the gradient. Ask us if you have any doubts!